learn/reference/tables.mdx

```diff
@@ -817,7 +817,7 @@ This can significantly simplify the construction of complex queries and reduce t
 
 ```javascript
 const leads = await LeadsTable.findRecords({
-  filter: AI`works either at ${workflow.company} OR is not in IT`,
+  filter: AI`works either at ${workflow.company} OR isn't in IT`,
```

Comment:
this was to make vale happy, but I think it's actually wrong/broken, so this blob will need a guard so vale doesn't complain.